.circleci/config.yml

```diff
@@ -111,3 +137,7 @@ workflows:
               php-version: [ '5.6', '7.0', '7.1', '7.2', '7.3', '7.4' ]
       - js-tests
       - e2e-tests
+  deploy:
+    jobs:
+      - svn-deploy:
+          context: genesis-svn
```

Comment:
@kienstra you'll want something in here so that it only deploys when tags are pushed. In AB we do this:

```
          filters:
            tags:
              only: /^\d+\.\d+\.\d+$/
            branches:
              ignore: /.*/
```

https://github.com/studiopress/atomic-blocks/blob/0d1808b57bcb4caf99ce6de419be1876fac7d02f/.circleci/config.yml#L280-L288

Reply:
Thanks, good idea!

Reply:
Applied in https://github.com/studiopress/genesis-custom-blocks/pull/32/commits/1a5eca83a7a8bd3e309c6ef9cf765beec26a204e